File: marcos006/programacao_basica/Questao14.java
import java.util.Scanner;

    public class Questao14 {
        public static void main(String[] args) {
            Scanner s;
            s = new Scanner(System.in);

        System.out.println("**Calculador de tempo de trabalho sem acidentes**");
        System.out.println("Informe o tempo em dias de trabalho sem acidentes: ");
        int d = s.nextInt();

        int a = d / 360;
        int m = (d - a *360) / 30;
        int dd = d - (a * 360) - m * 30;

        System.out.println("O tempo sem acidentes é " + a + " anos " + m + " mêses " + dd + " dias");

    }
}

File: marcos006/programacao_basica/questao14.py
print("Calculador de tempo de trabalho sem acidentes")
d = int(input("Informe o tempo em dias de trabalho sem acidentes: "))
a = d // 360
m = (d - a *360) // 30
dd = d - (a * 360) - m * 30
print("O tempo sem acidentes é " + str(a) + " anos " + str(m) + " mêses " + str(dd) + " dias")
